lib/types.ts

```diff
@@ -318,6 +318,11 @@ type Mapping<TKey extends OnyxKey> = ConnectOptions<TKey> & {
  */
 type OnyxInputValue<TOnyxValue> = TOnyxValue | null;
 
+/**
+ * Represents an Onyx collection input, that can be either a record of `TOnyxValue`s or `null` if the key should be deleted.
+ */
+type OnyxCollectionInput<TOnyxValue> = OnyxInputValue<Record<string, TOnyxValue | null>>;
```

Comment:
```suggestion
type OnyxCollectionInputValue<TOnyxValue> = OnyxInputValue<Record<string, TOnyxValue | null>>;
```

Just to be aligned with `OnyxInputValue` pattern